netci.groovy

```diff
@@ -17,7 +17,8 @@ def static getOSGroup(def os) {
         'Windows_NT':'Windows_NT',
         'FreeBSD':'FreeBSD',
         'CentOS7.1': 'Linux',
-        'OpenSUSE13.2': 'Linux']
+        'OpenSUSE13.2': 'Linux',
+        'LinuxARMEmulator': 'Linux']
```

Comment:
You;'re goiing to need to add LinuxArmEmulator to the osList on line 32ish.  You have this map right, but it's there to map between OS's and the OSGroup which is used when building mscorlib.  I suppose we could use the keys in the map as the OS list.  I'll have to look into that.